src/status_im/contexts/wallet/events.cljs

```diff
@@ -439,3 +439,32 @@
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
                 :duration 10000}]]])})))
+
+(rf/reg-event-fx
+ :wallet/save-address
+ (fn [_ [{:keys [address name color-id on-success on-error chain-short-names ens is-test]
+         :or   {on-success        (fn [])
+                on-error          (fn [])
+                name              ""
+                ens               ""
+                is-test           false
+                chain-short-names "eth:"}}]]
```

Comment:
@smohamedjavid @OmarBasem - do we have a consistent name for these chain-short-names?

Reply:
No, I mean the keys. It's not a good idea to reference the same concept as multiple different things. Consistent language helps a lot!

Reply:
In general we call it `network-short-name`, but in this context I think it is better to call it `address-prefix`

Reply:
Hmm, that's a bit generic. It should indicate the use case imo.
Like network-preference-short-names or just network-preferences.
I think that's what we have it in the db model. I'll look it up shortly